website/versioned_docs/version-0.64/rnm-getting-started.md

```diff
@@ -0,0 +1,51 @@
+---
+id: version-0.64-rnm-getting-started
```

Comment:
I think you will need to either update or delete the other rnm-getting-started files in 0.65+, so that we fall back to the 0.64 version (please check locally)

Reply:
How would I validate that deleting that file loads the right page on the website?

Reply:
clone the repo locally, make the changes, then run yarn start in the website folder (it will navigate to the local webserver)

Reply:
Thanks got it sorted. Locally it links to the 0.64 pages now.